UnitTests/GitCommands.Tests/Submodules/SubmoduleStatusProviderTests.cs

```diff
@@ -483,17 +467,10 @@ public async Task Submodule_status_changes_for_first_nested_module_with_second_n
 
             // Revert the change
             File.Delete(Path.Combine(_repo3Module.WorkingDir, "test.txt"));
-            changedFiles = GetStatusChangedFiles(currentModule);
-            changedFiles.Should().HaveCount(0);
-            await SubmoduleTestHelpers.UpdateSubmoduleStatusAndWaitForResultAsync(_provider, currentModule, changedFiles);
-            result.AllSubmodules.All(i => i.Detailed == null).Should().BeTrue();
-            result.OurSubmodules[0].Should().BeEquivalentTo(result.AllSubmodules[1]);
-
-            // Revert manually - not changed back automatically
-            result.TopProject.Detailed.IsDirty.Should().BeTrue();
```

Comment:
This has assertion has been removed without replacement. Applies to further testcases below.

Reply:
result.OurSubmodules.Should().BeEquivalentTo(result.AllSubmodules);
andsimilar is testedwhen result iscreated, no need to test it again